### ISSUE TITLE
Wavebox takes minutes to load gmail

### ISSUE THREAD

**Issue by CheechGe** [wavebox/waveboxapp#939]
* Wavebox Version: [e.g. 3.0.0] 
4.7.1
* Operating System & Version: [e.g. Windows 10, Mac OSX 10.3.4, Ubuntu 17.10]
Windows 10
* (Linux only) Install method: [e.g. Debian installer, Snap Package, Tar, Apt-Repo, etc...]
* Account type (if applicable): [e.g. Gmail, Office 365, Trello, etc...]
Gmail
**Expected & actual behavior**
Some information about what you expect to happen and what actually happens.
Since this latest release it sometimes takes 5 minutes to load my gmail account.  After it loads the first time if WB is minimized and then maximized it goes through the same process.  this only started since this latest update.  After WB is maximized the only thing that appears is the menu bar.  The rest of the screen is blank until it finally loads up.
![wb_01](https://user-images.githubusercontent.com/3653639/52861707-50182a00-3101-11e9-9528-1d37e7e9def0.png)

**Steps to reproduce**
Are there any steps that we can take to reproduce the bug?
explained above

**Is the bug persistent or intermittent?**
Do you only see it sometimes or does it always happen?
Everytime
**Screenshots**
Screenshots often help us narrow down a bug more easily.

**Additional information**

**Comment by Thomas101** [comment]
Thanks for reporting, I think this is an issue that causes the window to not repaint correctly on some machine configurations. We've got a proper fix coming for this our beta channel shortly.

In the meantime there are a couple of things to try, each in turn as the first one of these normally works...

* Under Settings > Advanced, enable `Force window repaint on restore`
* Under Settings > Advanced, disable `Hardware Acceleration`

Failing that, does if it happens again, does resizing the window make it render again?

**Comment by CheechGe** [comment]
Hi Thomas

I tried your first 2 suggestions independently and neither of them worked
so I set them back to their original settings.  Resizing the window does
cause the window to repaint.  After I submitted the bug report I had
noticed that playing around with the window did cause it to repaint.  I
thought it happened when I just moved the window but it is possible that I
had inadvertently resized it in the process.

Thanks,

Frank
